pkg/reader/ccip.go

```diff
@@ -550,6 +551,63 @@ func (r *ccipChainReader) GetChainFeePriceUpdate(ctx context.Context, selectors
 	return feeUpdates
 }
 
+func (r *ccipChainReader) GetRMNRemoteConfig(
+	ctx context.Context,
+	destChainSelector cciptypes.ChainSelector,
+) (rmntypes.RMNRemoteConfig, error) {
+	if err := validateExtendedReaderExistence(r.contractReaders, destChainSelector); err != nil {
+		return rmntypes.RMNRemoteConfig{}, err
+	}
+
+	contractAddress, err := r.GetContractAddress(consts.ContractNameRMNRemote, destChainSelector)
+	if err != nil {
+		return rmntypes.RMNRemoteConfig{}, fmt.Errorf("get RMNRemote contract address: %w", err)
+	}
+
+	var vc versionnedConfig
+	err = r.contractReaders[destChainSelector].ExtendedGetLatestValue(
+		ctx,
+		consts.ContractNameRMNRemote,
+		consts.MethodNameGetVersionedConfig,
+		primitives.Unconfirmed,
+		map[string]any{},
+		&vc,
+	)
+	if err != nil {
+		return rmntypes.RMNRemoteConfig{}, fmt.Errorf("failed to get RMNRemote config: %w", err)
```

Comment:
General tip. Don't say `failed to` in the errors.
Otherwise with wrapping it becomes:
```
failed to do this: failed to do that: failed to foo
```

we can just do
```
do this: do that: foo
```

Reply:
Makes sense, thanks for the tips

Reply:
@dimkouv I wonder if we can create a linter just for that.

Reply:
I can look into it as a side project 😛